e2e/Makefile

```diff
@@ -0,0 +1,26 @@
+test-e2e: test-e2e-minimal test-e2e-individually test-e2e-complete
```

Comment:
Ah okay I think I see what you are saying. Hmmm. I guess ideally it would be nice to call it once but while working locally I think it is convenient that we are always using the latest build.

I don't mind with leaving it as is. The build step is fairly speedy anyway.

Reply:
yeah you're right, the build step (ts compilation) is pretty quick, especially compared to installing node modules. happy to leave it as is 😁